templates/profile.html

```diff
@@ -24,9 +29,13 @@
         <h2 id="username-heading">{{ displayed_user.username }}</h2>
         <h2>Rank {{ rank }}</h2>
         <h2>{{ score }} Points</h2>
+        <p>Drag/pinch, or hold shift and use mouse wheel/drag to zoom/pan</p>
     </section>
 
     <div id="chart-container">
+        <div class="chart-toolbar">
+            <button onclick="resetChartZoom()" title="Reset zoom">{{ icons.zoom__fit() }}</button>
```

Comment:
Should we add the fullscreen chart option for the profile page? It doesn't really make too much sense here.